.editorconfig

```diff
@@ -149,3 +149,9 @@ dotnet_diagnostic.CA2254.severity = suggestion
 
 # Extract strings to resource table (the solution will not be localized).
 dotnet_diagnostic.CA1303.severity = none
+
+# Use compound assignment
+dotnet_diagnostic.IDE0074.severity = suggestion
+
+# Nullable directive is redundant
+dotnet_diagnostic.IDE0240.severity = suggestion
```

Comment:
These lines are needed to get compilation with Visual Studio 2022 (Preview) and MSBuild 17.3.0.33003